raiden/tests/integration/network/proxies/test_token_network_registry.py

```diff
@@ -45,7 +48,21 @@ def test_token_network_registry(
         token_name='TKN',
         token_symbol='TKN',
     )
+
     test_token_address = to_canonical_address(test_token.contract.address)
+    # try to register a token network not following ERC20 protocol
+
+    def unimplemented_total_supply(block_identifier='latest'):  # pylint: disable=unused-argument
+        return ''
+
+    with patch.object(Token, 'total_supply', side_effect=unimplemented_total_supply):
```

Comment:
Nitpick: you can also just use `return_value=''` here instead of defining a `side_effect`.

Reply:
Oh that's a good tip thank you. Will add it.